haystack/pipelines/base.py

```diff
@@ -1162,6 +1162,7 @@ def eval(
                         sas_model_name_or_path=sas_model_name_or_path,
                         batch_size=sas_batch_size,
                         use_gpu=sas_use_gpu,
+                        use_auth_token=None,
```

Comment:
Why pass `None` here? use_auth_token is not a required parameter.

Reply:
In my opinion it's fine to add it to the `eval` method. As it is an optional parameter it isn't a breaking change.